tests/sles4sap/sap_deployment_automation_framework/connect_to_deployer.pm

```diff
@@ -9,30 +9,40 @@ use parent 'sles4sap::sap_deployment_automation_framework::basetest';
 use strict;
 use warnings;
 use testapi;
-use sles4sap::sap_deployment_automation_framework::deployment;
-use sles4sap::console_redirection;
+use sles4sap::sap_deployment_automation_framework::deployment
+  qw(serial_console_diag_banner
+  az_login
+  sdaf_prepare_ssh_keys
+  );
+use sles4sap::sap_deployment_automation_framework::deployment_connector
+  qw(get_deployer_vm
+  get_deployer_ip
+  );
+use sles4sap::console_redirection qw(redirection_init);
 use serial_terminal qw(select_serial_terminal);
 
 sub test_flags {
     return {fatal => 1};
 }
 sub run {
-
     select_serial_terminal();
     serial_console_diag_banner('Module sdaf_redirect_console_to_deployer.pm : start');
+    az_login();
 
-    # autossh is required for console redirection to work
-    assert_script_run('zypper in -y autossh');
+    my $deployer_vm_name = get_deployer_vm;
+    die 'Deployer VM not found' unless $deployer_vm_name;
```

Comment:
and it should... right? should be created by [create_deployer_vm.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19654/files#diff-641604be24c2a28531e5c247b07e006828a64e329a640490f343ad37e8001bad)
maybe add a code comment about it

Reply:
Done.